electron/mainRenderer.js

```diff
@@ -600,15 +614,28 @@ function resizeWindow() {
 }
 
 function populateDeck(elem) {
-    deckID = elem.getAttribute('data-deckid')
-    $.each(appData.player_decks, (i, v) => {
-        if (v.deck_id == deckID) {
-            appData.selected_list = v.cards;
-            appData.selected_list_name = v.pool_name;
-            appData.list_selected = true;
-            appData.no_list_selected = false;
-        }
-    })
+    deckID = elem.getAttribute('data-deckid');
+    appData.activeDeck = deckID;
+    deck = getDeckById(deckID);
+
+    const types = ['daily','alltime'];
+    $.each(types, (i, type) => {
+      if (appData.winLossObj[type][deckID] === undefined) {
+        appData.winLossObj[type][deckID] = {win: 0, loss: 0, name: deck.pool_name}
+      }
+      const counter_prefix = type === 'daily' ? 'dailyD' : 'd';
```

Comment:
This isn't blocking, but I would much rather prefer `const counterName = type === 'daily' ? 'dailyDeckWinCounter' : 'deckWinCounter';` I get what's going on here, but it seems unnecessarily golfy and is kinda opaque on first glance.

Reply:
Yanno, it was really late when I wrote that. I had some reason to go that way and I know I came back from something, so it kinda got left in a weird state. Yeah, that should look better. Sorry.